appserver/pom.xml

```diff
@@ -883,12 +883,12 @@
                             <artifact>
                                 <groupId>jakarta.servlet</groupId>
                                 <artifactId>jakarta.servlet-api</artifactId>
-                                <version>${servlet-api.version}</version>
```

Comment:
I wonder why this missing property didn't bother the plugin...

Reply:
Seems it does [master:1488](https://ci.eclipse.org/glassfish/job/glassfish_build-and-test-using-jenkinsfile/job/master/1488/):
```
[2024-07-07T10:38:50.796Z] [ jakarta.servlet:jakarta.servlet-api:6.0.0 ]
[2024-07-07T10:38:50.796Z] { groupIdPrefix=jakarta. API final spec-impl-version=null }
[2024-07-07T10:38:50.796Z] - ERROR: missing configuration ( spec-version api-package spec-impl-version )
```
but not enough to cause failure.